### PR TITLE
[0.9] 0.9.5

### PR DESCRIPTION
**Pull request by nabobalis**
PR for 0.9.5

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "astropy_helpers"]
 	path = astropy_helpers
-	url = https://github.com/Cadair/astropy-helpers.git
+	url = https://github.com/astropy/astropy-helpers.git
```
Comment (nabobalis) [inline .gitmodules]: THIS HAS BEEN SO MUCH PAIN FOR ME.

**Comment by nabobalis** [comment]
ITS FINE JUST THAT GOES TEST

File: .gitmodules (hunk 1 above)
Comment (Cadair) [reply .gitmodules]: oh man didn't we back this out already :scream_cat:

Comment (nabobalis) [reply .gitmodules]: Yeah, just on this branch.

**Pull request merged**